LongWarOfTheChosen/Config/XComGameData_WeaponData.ini

```diff
@@ -1768,3 +1768,210 @@ SPARK_NOVA_SELF_BASEDAMAGE= (Damage=0, Spread = 0, PlusOne = 0, Crit = 0, Pierce
+
+;Ripjack stats
+WRISTBLADE_MAGNETIC_BASEDAMAGE=(Damage=7, Spread=0, PlusOne=0, Crit=2, Pierce=0, Shred=0, Tag="", DamageType="Melee")
+WRISTBLADE_BEAM_BASEDAMAGE=(Damage=10, Spread=0, PlusOne=0, Crit=3, Pierce=0, Shred=0, Tag="", DamageType="Melee")
+
+WRISTBLADE_MAGNETIC_AIM = 20
+WRISTBLADE_MAGNETIC_CRITCHANCE = 0
+WRISTBLADE_MAGNETIC_ICLIPSIZE = 0
+WRISTBLADE_MAGNETIC_ISOUNDRANGE = 8
+WRISTBLADE_MAGNETIC_IENVIRONMENTDAMAGE = 1
+WRISTBLADE_MAGNETIC_STUNCHANCE = 25
+
+WRISTBLADE_BEAM_AIM = 20
+WRISTBLADE_BEAM_CRITCHANCE = 0
+WRISTBLADE_BEAM_ICLIPSIZE = 0
+WRISTBLADE_BEAM_ISOUNDRANGE = 8
+WRISTBLADE_BEAM_IENVIRONMENTDAMAGE = 1
+
+;Shard Gauntlet stats
+SHARDGAUNTLET_MAGNETIC_BASEDAMAGE=(Damage=6, Spread=0, PlusOne=50, Crit=2, Pierce=0, Shred=0, Tag="", DamageType="Melee")
+SHARDGAUNTLET_BEAM_BASEDAMAGE=(Damage=10, Spread=0, PlusOne=50, Crit=2, Pierce=0, Shred=0, Tag="", DamageType="Melee")
++SHARDGAUNTLET_MAGNETIC_EXTRADAMAGE=(Damage=5, Spread=1, PlusOne=34, Crit=0, Pierce=0, Tag="Volt", DamageType="Psi")
++SHARDGAUNTLET_BEAM_EXTRADAMAGE=(Damage=7, Spread=1, PlusOne=34, Crit=0, Pierce=0, Tag="Volt", DamageType="Psi")
++SHARDGAUNTLET_MAGNETIC_EXTRADAMAGE=(Damage=8, Spread=1, PlusOne=67, Crit=0, Pierce=0, Tag="Volt_Psi", DamageType="Psi")
++SHARDGAUNTLET_BEAM_EXTRADAMAGE=(Damage=11, Spread=1, PlusOne=67, Crit=0, Pierce=0, Tag="Volt_Psi", DamageType="Psi")
++SHARDGAUNTLET_MAGNETIC_EXTRADAMAGE=(Damage=4, Spread=1, PlusOne=0, Crit=0, Pierce=0, Tag="IonicStorm", DamageType="Psi")
++SHARDGAUNTLET_BEAM_EXTRADAMAGE=(Damage=6, Spread=1, PlusOne=0, Crit=0, Pierce=0, Tag="IonicStorm", DamageType="Psi")
++SHARDGAUNTLET_MAGNETIC_EXTRADAMAGE=(Damage=6, Spread=1, PlusOne=0, Crit=0, Pierce=0, Tag="IonicStorm_Psi", DamageType="Psi")
++SHARDGAUNTLET_BEAM_EXTRADAMAGE=(Damage=9, Spread=1, PlusOne=0, Crit=0, Pierce=0, Tag="IonicStorm_Psi", DamageType="Psi")
+
+SHARDGAUNTLET_MAGNETIC_AIM = 20
+SHARDGAUNTLET_MAGNETIC_CRITCHANCE = 10
+SHARDGAUNTLET_MAGNETIC_ICLIPSIZE = 0
+SHARDGAUNTLET_MAGNETIC_ISOUNDRANGE = 8
+SHARDGAUNTLET_MAGNETIC_IENVIRONMENTDAMAGE = 1
+SHARDGAUNTLET_MAGNETIC_STUNCHANCE = 25
+
+SHARDGAUNTLET_BEAM_AIM = 20
+SHARDGAUNTLET_BEAM_CRITCHANCE = 10
+SHARDGAUNTLET_BEAM_ICLIPSIZE = 0
+SHARDGAUNTLET_BEAM_ISOUNDRANGE = 8
+SHARDGAUNTLET_BEAM_IENVIRONMENTDAMAGE = 1
+
+;Autopistol stats
+SIDEARM_MAGNETIC_BASEDAMAGE=(Damage=4, Spread=0, PlusOne=50, Crit=1, Pierce=0, Shred=0, Tag="", DamageType="Projectile_MagXCom")
+SIDEARM_BEAM_BASEDAMAGE=(Damage=6, Spread=1, PlusOne=50, Crit=2, Pierce=0, Shred=0, Tag="", Rupture=1, DamageType="Projectile_BeamXCom")
+
+SIDEARM_MAGNETIC_AIM = 0
+SIDEARM_MAGNETIC_CRITCHANCE = 0
+SIDEARM_MAGNETIC_ICLIPSIZE = 99
+SIDEARM_MAGNETIC_ISOUNDRANGE = 15
+SIDEARM_MAGNETIC_IENVIRONMENTDAMAGE = 1
+
+SIDEARM_BEAM_AIM = 0
+SIDEARM_BEAM_CRITCHANCE = 0
+SIDEARM_BEAM_ICLIPSIZE = 99
+SIDEARM_BEAM_ISOUNDRANGE = 15
+SIDEARM_BEAM_IENVIRONMENTDAMAGE = 1
+
+;Weapon Range Tables
+VEKTOR_MAGNETIC_RANGE[0] = 0
+VEKTOR_MAGNETIC_RANGE[1] = 26
```

Comment:
Why 0 for point blank? Seems weird. Is this based off of vanilla numbers?

Reply:
Or is zero not point blank? It means "on top of"?

Reply:
No this is just an oopsie on my part. Didn't see it was at zero. I'll fix this.

Reply:
Fixed this. Now Vektor at point blank is 30.